eng/pipelines/templates/jobs/python-analyze-weekly.yml

```diff
@@ -0,0 +1,86 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: TestMarkArgument
```

Comment:
I believe that `TestMarkArgument` is passed through to `pytest` via `tox <blah> -- -m <mark_arg>`. (Used in cosmos for `not cosmosEmulator`)

I don't think that applies to these checks?

Reply:
You're right, it's not used. I removed it.